torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -70,6 +75,20 @@ def forward(
         **kwargs,
     ) -> Explanation:
 
+        # Check if the method supports multiple indices.
+        if isinstance(index, list) and len(index) > 1:
+            if not self._support_multiple_indices():
+                raise ValueError(
+                    f"{self.attribution_method.__name__} does not "
+                    "support multiple indices. Please use a single "
+                    "index or a different method.")
+            if self.kwargs.get('internal_batch_size') != 1:
+                warnings.warn("Overriding 'internal_batch_size' to 1")
+                self.kwargs['internal_batch_size'] = 1
+            warnings.warn(
+                "For all indices, the explanations are computed on the"
+                "same mask. This can lead to overlapping explanations.")
+
```

Comment:
That would be the case for every explainer, so I am not sure I understand the warning here.